js/app/page.tsx

```diff
@@ -176,19 +176,23 @@ function TopBar() {
         </Badge>
       )}
       {isDemoSite && prURL && demoPrId && (
-        <>
-          <Badge fontSize="sm" color="white/80" variant="outline">
-            <HStack>
-              <Box>demo mode</Box>
-              <Box borderLeft="1px" borderLeftColor="whiteAlpha.500" paddingLeft="8px">
-                <Link href={prURL} _hover={{ textDecoration: "none" }} target="_blank">
-                  <Icon as={VscGitPullRequest} boxSize="3" fontWeight="extrabold" strokeWidth="1" />
-                  {` #${demoPrId}`} <LuExternalLink />
-                </Link>
-              </Box>
-            </HStack>
-          </Badge>
-        </>
+        <Badge fontSize="sm" color="white/80" variant="outline" textTransform="uppercase">
+          <HStack>
+            <Box>demo mode</Box>
+            <Box borderLeftWidth="1px" borderLeftColor="white/80" paddingLeft="8px">
+              <Link href={prURL} _hover={{ textDecoration: "none" }} target="_blank">
+                <Icon
+                  color="white/80"
+                  as={VscGitPullRequest}
+                  boxSize="3"
+                  fontWeight="extrabold"
+                  strokeWidth="1"
+                />
+                <Text color="white/80">{` #${demoPrId}`}</Text>
```

Comment:
you can remove this awkward whitespacing `{` #${demoPrId}`}`, by adding margin right to the Icon:

```typescript
<Icon
  color="white/80"
  as={VscGitPullRequest}
  boxSize="3"
  marginRight={1}
  fontWeight="extrabold"
  strokeWidth="1"
/>
<Text color="white/80">#{demoPrId}</Text>
```